bridge-ui/src/app/faq/page.tsx

```diff
@@ -0,0 +1,214 @@
+            <Link
+              href="https://support.metamask.io/managing-my-tokens/moving-your-tokens/funds-sent-on-wrong-network/#non-evm-tokens-and-networks"
+              rel="noopener noreferrer"
+              passHref
+              target="_blank"
+              className="link link-primary"
+            >
+              MetaMask Support article
+            </Link>{" "}
+            for an explanation). This means you cannot bridge to Bitcoin or Solana, for example, using the Linea Bridge.
+          </p>
+        </Collapse>
+
+        <Collapse title="I need to claim a transaction. What do I do?">
+          <p>
+            If you chose manual claiming, you need to return to the Linea Bridge and make an additional transaction. It
+            typically takes around 20 minutes (two Ethereum epochs) before you can claim a deposit; withdrawals from
+            Linea can take much longer, and vary in duration.
+          </p>
+          <p>
+            To claim, head to the &apos;Transactions&apos; tab and locate the bridge transaction you want to claim; it
+            should be marked as &apos;Ready to claim&apos;. Click on it, and then on the &apos;Claim&apos; button to
+            initiate the transaction.
+          </p>
+        </Collapse>
+
+        <Collapse title="What fees will I pay?">
+          <p>
+            Linea does not charge fees for using the bridge. However, you will need to pay gas fees for transactions.
+            Gas fees vary according to network activity, so we cannot accurately predict how much you&apos;ll pay. The
+            bulk of the cost will be for interacting with Ethereum Mainnet; Linea&apos;s gas fees are considerably
+            cheaper.
+          </p>
+          <p>
+            If you use manual claiming, you pay gas fees for the bridge transaction and then gas fees for the claim
+            transaction.
+          </p>
+          <p>
+            For automatic claiming, you pay gas fees for your initial bridge transaction and then a fee to cover the gas
+            costs of the postman executing the claim on your behalf.
+          </p>
+        </Collapse>
+
+        <Collapse title="Why can I only select Linea Mainnet and Ethereum Mainnet?">
+          <p>
+            The Linea Bridge is designed to transact between layer 1 (Ethereum Mainnet) and layer 2 (Linea Mainnet),
+            rather than for bridging to any other chains.
+          </p>
+          <p>
+            If you want to bridge to other EVM-compatible networks, consider using the{" "}
+            <Link
+              href="https://portfolio.metamask.io/bridge"
+              rel="noopener noreferrer"
+              passHref
+              target="_blank"
+              className="link link-primary"
+            >
+              MetaMask Portfolio bridge
+            </Link>
+            .
+          </p>
+        </Collapse>
+
+        <Collapse title="I can't see the token I want to bridge">
```

Comment:
this isn't phrased as a question  - `Why can't I see the token I want to bridge?`